src/assets/news/2024-06-07 - Lemmy Release v0.19.4 - Image Proxying and Federation improvements.md

```diff
@@ -0,0 +1,119 @@
+
+Reliability and security of federation have been improved, and numerous bugs squashed. Signed fetch was broken and is fixed now.
+
+### Vote display user setting
+
+There is now a [user setting to change the way vote counts are displayed](https://github.com/LemmyNet/lemmy/pull/4450), called [vote display mode](https://join-lemmy.org/api/interfaces/LocalUserVoteDisplayMode.html).
+
+You can specify which of the following vote data you'd like to see (or hide): Upvotes, Downvotes, Score, Upvote Percentage, or none of the above. The default (based on user feedback) is showing the upvotes + downvotes.
+
+App developers will need to update their apps to support this setting.
+
+### RSS Feeds
+
+RSS feeds now include [post thumbnail](https://github.com/LemmyNet/lemmy/pull/4413) and [embedded images](https://github.com/LemmyNet/lemmy/pull/4442).
+
+### Security Audit
+
+A security audit was recently performed on Lemmy. Big thanks to [Radically Open Security](https://www.radicallyopensecurity.com/) for the generous funding, and to Sabrina Deibe and Joe Neeman for carrying out the audit. The focus was on federation logic, and discovered various problems in this area. Most of the problems are being mitigated as part of this release. Fortunately no critical security vulnerabilities were discovered.
+
+This is already the third security audit of Lemmy, all organized by ROS. We're greatly indebted to them for their support.
+
+### Other Changes
+
+- [Added Community `local_subscribers` count](https://github.com/LemmyNet/lemmy/pull/4166)
+- [Support for custom post thumbnail](https://github.com/LemmyNet/lemmy/pull/4425)
+- For new user accounts the [interface language](https://github.com/LemmyNet/lemmy/pull/4435) and [discussion languages](https://github.com/LemmyNet/lemmy/pull/4550) are set automatically based on `accept-language` HTTP header
+- [Added instance-level default sort type](https://github.com/LemmyNet/lemmy/pull/4454)
+- [Indicate to user when they are banned from community](https://github.com/LemmyNet/lemmy/pull/4458)
+- [Added alt_text for image posts](https://github.com/LemmyNet/lemmy/pull/4477)
+- [Dont require leading ! or @ to fetch a user or community](https://github.com/LemmyNet/lemmy/pull/4513)
+- [Extra fields for PostReport and CommentReport views](https://github.com/LemmyNet/lemmy/pull/4520)
+
+### Full Changelog
+
+- [API changes - lemmy-js-client 0.19.3 -> 0.19.4](https://github.com/LemmyNet/lemmy-js-client/compare/0.19.3-alpha.1...0.19.4)
+- [Lemmy Backend](https://github.com/LemmyNet/lemmy/pulls?q=is%3Apr+merged%3A%3E2024-01-22)
+- [Lemmy-UI](https://github.com/LemmyNet/lemmy-ui/pulls?q=is%3Apr+merged%3A%3E2024-01-22)
```

Comment:
Could include the lemmy rust client here as well once that is updated to 0.19.4.

Reply:
K, I'll add it.